miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/LibraryDto.java

```diff
@@ -51,6 +51,7 @@
   private SampleDto sample;
   private String receivedDate;
   private String sampleBoxPositionLabel;
+  private String defaultTargetedSequencing;
```

Comment:
Why is this being added to the library when `targetedSequencing` is a dilution attribute?